examples/gallery/line/line-custom-cpt.py

```diff
@@ -0,0 +1,31 @@
+"""
+Line colors with a custom CPT
+-----------------------------
+
+The color of the lines made by :meth:`pygmt.Figure.plot` can be set according to a
+custom CPT and assigned with the *pen* argument.
+
+The custom CPT can be used by setting the plot command's `cmap` argument to *True*. The
+`zvalue` argument sets the z-value (color) to be used from the custom CPT, and the line
+color is set as the z-value by using **+z** when setting the `pen` color.
```

Comment:
Following the convention in #631.
```suggestion
The custom CPT can be used by setting the plot command's ``cmap`` argument to ``True``. The
``zvalue`` argument sets the z-value (color) to be used from the custom CPT, and the line
color is set as the z-value by using **+z** when setting the ``pen`` color.
```